src/charm.py

```diff
@@ -38,20 +43,19 @@ class UpdateRunnerBinEvent(EventBase):
     """Event representing a periodic check for new versions of the runner binary."""
 
 
-class GithubRunnerOperator(CharmBase):
+class GithubRunnerCharm(CharmBase):
     """Charm for managing GitHub self-hosted runners.
 
     TODO:
-    * Remove support of LXD containers, due to security concerns.
     * Review the action results fields, and use TypedDict for the action results, and JSON-like
         return values.
     """
 
     _stored = StoredState()
 
-    def __init__(self, *args) -> None:
+    def __init__(self, *args, **kargs) -> None:
```

Comment:
I'm surprised that `flake8-docstrings-complete` is not complaining about the undocumented arguments here?

Reply:
Since `args` and `kargs` are standard convention, maybe `flake8-docstrings-complete` does not complain about it?

Reply:
I wrote the linter 😉  I can see that it is being used, I'll look into why it wasn't caught when I get some time. In the meantime, do you think it is worth documenting them in the docstring?

Reply:
Will do